concepts/60 Themes and Styles/05 Predefined Themes/60 Switch Between Themes at Runtime/2 Without Page Reload.md

```diff
@@ -20,7 +20,11 @@ You can use this approach only if the themes belong to the same group. For insta
             <!-- ... -->
         </head>
 
-1. Switch to a theme using the [DevExpress.ui.themes.current(themeName)](/api-reference/50%20Common/utils/ui/themes/current(themeName).md '/Documentation/ApiReference/Common/utils/ui/themes/#currentthemeName') method. It accepts the `data-theme` attribute's value from the previous code. If you use SVG components, you should also call the [refreshTheme()](/api-reference/50%20Common/utils/viz/refreshTheme().md '/Documentation/ApiReference/Common/Utils/viz/#refreshTheme') method to update their theme. The following example shows how to apply the Generic Contrast theme:
+1. Use the [DevExpress.ui.themes.current(themeName)](/api-reference/50%20Common/utils/ui/themes/current(themeName).md '/Documentation/ApiReference/Common/utils/ui/themes/#currentthemeName') method to switch to another theme. 
+
+The method accepts the `data-theme` attribute's value. For example, if you want to switch to `dx.generic.dark`, use the `generic.dark` (without the 'dx' prefix) as an argument in the method. For a custom theme, specify the theme you used as a base.
```

Comment:
```suggestion
The method accepts the `data-theme` attribute value. For example, if you want to switch to `dx.generic.dark`, pass `generic.dark` (without the 'dx' prefix) as an argument in the method. For a custom theme, specify the theme you used as a base.
```

Reply:
Since you're talking about an argument, note that I changed "use" to "pass". Is it technically correct, or is "use" better?

Reply:
"Pass" is okay, didn't think of it. Thanks!